aries-site/src/pages/templates/forms.mdx

```diff
@@ -23,105 +41,73 @@ import { Example } from '../../layouts';
-- There may be cases where optional fields may be appropriate. In these forms, clearly denote which fields are optional vs. required. This way, users won't be forced to infer anything - but can instead stay focused on completing their task.
-- Limit the number of optional fields as much as possible. Remember that the more fields you have on your form, the more you risk overwhelming the users.
+**Format and fields**
+- User-entered input must be encoded to prevent [SQL injection](https://www.w3schools.com/sql/sql_injection.asp) and [XSS attacks](https://en.wikipedia.org/wiki/Cross-site_scripting).
+- Wrap all input elements within a FormField `<FormField label="Username"><TextInput /></FormField>`  
+- Reveal fields progressively so the user is not overwhelmed 
+- Write in sentence case (capitalize the first letter and all following letters should be lowercase)  
+- Fill in default values for as many fields as possible 
+- Use clear, concise title labels for each field 
+- Write helpful placeholder text inside the fields 
+- If the user would likely have doubts, include helpful text below the field, such as [password requirements](#signup) or [additional resources](#customize) on a field's context
+- Place the submit, reset and cancel buttons at the bottom left of the form. [See responsive button behavior](https://design-system.hpe.design/components/button?q=button#responsiveness). 
+- Use specific labels on buttons that tell the user what action to expect 
 
-<ExampleImagePreview
-  label="Required and Optional Fields Example"
-  src="/required&optional-Fields-example.png"
-  href="#required-and-optional-fields"
-  alt="HPE Design System Form example where required and optional fields are used."
-/>
-
-**Forms where 'asterisk' and 'optional' can be skipped**
+**Actions after the user submits information**
+- Provide feedback when the form is submitted successfully (e.g. notification, success page, green check mark, etc.) 
+- [In the case of an error](#requiredoptional), red highlight the field where the error occurred 
+- List below the button all relevant errors and the corrective action the user must take 
+- Allow the user to continue navigating through the application immediately
 
-A login/registration form, which typically shows two to three required fields do not need to be annotated in any way - the form is short and the necessity of completing the fields is relatively obvious.
 
-For login/registration forms, apply `required={{ indicator: false }}` to the FormFields to remove the required indicator.
+## Don'ts
 
-<ExampleImagePreview
-  label="Forms where 'asterisk' and 'optional' can be skipped"
-  src="/form-example-nonrequired.png"
-  href="#sign-in"
-  alt="HPE Design System Form where required and optional can be skipped"
-/>
+**Required fields**
+- Have more than two optional fields 
```

Comment:
This seems like we shouldn't have more than two optional fields in an initial task flow including a form (such as sign-up, etc.) -- but we also talk about how we can support more than 2 optional fields in something like profile settings (location, role, preferred contact method, phone number, secondary email, etc.). Should we clarify?

Reply:
Also this bullet point doesn't seem to make sense under "Required Fields" section

Reply:
Also not sure if want to limit users not to have options ( it is use case base ..

Reply:
I moved the "more than 2" to the "Fields" category below. And, again, the Do's and Don'ts are the ideal scenario. Any "exceptions," "edge cases," etc. can be addressed in the "Exceptions" section below.